examples/md-flexible/src/TypeDefinitions.h

```diff
@@ -50,16 +50,33 @@ using ParticleType = mdLib::MultisiteMoleculeLJ;
 using ParticleType = mdLib::MoleculeLJ;
 #endif
 
+namespace mdFlexibleTypeDefs {
+/**
+ * If AutoPas is compiled with FLOP logging enabled, use functors with FLOP counting enabled.
+ */
+constexpr bool countFLOPs =
+#ifdef AUTOPAS_LOG_FLOPS
+    true;
+#else
+    false;
+#endif
+}  // namespace mdFlexibleTypeDefs
+
 #if defined(MD_FLEXIBLE_FUNCTOR_AUTOVEC)
 /**
  * Type of LJFunctorTypeAutovec used in md-flexible.
  * Switches between mdLib::LJFunctor and mdLib::LJMultisiteFunctor as determined by CMake flag
  * MD_FLEXIBLE_MODE.
  */
 #if MD_FLEXIBLE_MODE == MULTISITE
-using LJFunctorTypeAutovec = mdLib::LJMultisiteFunctor<ParticleType, true, true>;
+using LJFunctorTypeAutovec = mdLib::LJMultisiteFunctor<ParticleType, true, true, autopas::FunctorN3Modes::Both, false,
+                                                       mdFlexibleTypeDefs::countFLOPs>;
+#ifdef AUTOPAS_LOG_FLOPS
+#warning "AUTOPAS_LOG_FLOPS enabled but is not suported for compiled functor (LJMultisiteFunctor)!"
```

Comment:
```suggestion
#pragma message "AUTOPAS_LOG_FLOPS enabled but is not suported for compiled functor (LJMultisiteFunctor)!"
```

In other places we used `#pragma message` over `#warning` because it seems to be supported more widely. AFAIK there is no universal compile time warning mechanism defined by the standard :/
Have you read something different recently?
If not please adapt all following `#warning` too.